core/src/main/java/com/ericsson/bss/cassandra/ecchronos/core/CASLockFactory.java

```diff
@@ -115,13 +109,15 @@ public final class CASLockFactory implements LockFactory, Closeable
     private final PreparedStatement myUpdateLockStatement;
     private final PreparedStatement myRemoveLockPriorityStatement;
     private final LockCache myLockCache;
-
+    private final long myLockUpdateTimeInSeconds;
+    private final int myFailedLockRetryAttempts;
     private CASLockFactory(final Builder builder)
```

Comment:
add newline between constructor and vars.

Reply:
done!